Introdução à Programação/Lista 4/Ex07.alg

```diff
@@ -0,0 +1,43 @@
+algoritmo "ex07"
+// Fun��o :
+// Autor :
+// Data : 7/6/2015
+// Se��o de Declara��es 
+var
+   op, idade:inteiro
+   sair, sexo:caracter
+inicio
+      op <- 1

```

Comment:
Não, pois as duas paradinhas têm que ser verdadeiras, pois se uma delas for falso, de acordo com o exercício, exige a saída do loop.